lib/utils/docker.coffee

```diff
@@ -128,7 +130,7 @@ exports.runBuild = (params, options, getBundleInfo) ->
 				buildSuccess: (image) ->
 					if options.tag?
 						console.log("Tagging image as #{options.tag}")
-					resolve(image)
+					resolve([image, logs])
```

Comment:
I think it'd be nicer to use `{ image, logs }` so that it's clear what each property means, rather than trying to map an index to a meaning (a similar idea to using `funcWithBoolOpts(true, false, false, true, false)` being much clearer with an object)